tools/migration/internal/logger_test.go

```diff
@@ -0,0 +1,75 @@
+package internal_test
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	. "github.com/filecoin-project/go-filecoin/tools/migration/internal"
+)
+
+func TestLoggerWritesToFile(t *testing.T) {
+	f, err := ioutil.TempFile("", "logfile")
+	require.NoError(t, err)
+	defer func() {
+		require.NoError(t, os.Remove(f.Name()))
+	}()
```

Comment:
Question: is it necessary or valuable to remove tempfiles, or files created in a tempdir? Isn't the whole point that they will be unlinked automatically when the process exits? This adds a bunch of noise to our tests which I am not confident is justified.

Reply:
I'm going off of the advice in [ioutil](https://golang.org/pkg/io/ioutil/#example_TempFile): "It is the caller's responsibility to remove the file when no longer needed."